scripts/modules/b2b-account/quotes.js

```diff
@@ -17,18 +17,18 @@ define([
     "modules/views-paging",
     'modules/editable-view',
     "modules/models-quotes",
-    "modules/b2b-account/account-address-search"], 
+    "modules/b2b-account/account-address-search",
+    "modules/b2b-account/email-quote"], 
     function ($, api, _, Hypr, Backbone, HyprLiveContext,
         CustomerModels, CartModels, B2BAccountModels, ProductModalViews,
         ProductPicker, ProductModels, WishlistModels, MozuGrid, MozuGridCollection,
```

Comment:
remove unnecessary models that we have injected here. Like i think 'ProductPicker' we haven't used in this file. but double-check and remove something which we haven't used here.